services/workspace_tabs.py

```
@@ -0,0 +1,477 @@
+                    ctx = json.loads(row["value"])
+                    layouts = ctx.get("projectLayouts")
+                    if isinstance(layouts, list):
+                        project_layouts_map.setdefault(cid, [])
+                        for layout in layouts:
+                            if isinstance(layout, str):
+                                try:
+                                    obj = json.loads(layout)
+                                    if isinstance(obj, dict) and obj.get("rootPath"):
+                                        project_layouts_map[cid].append(obj["rootPath"])
+                                except Exception:
+                                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)